### PR TITLE
🚨🚨✨ Source Facebook Pages: Upgrade Facebook API to v19.0

### PR DESCRIPTION
**Pull request by tolik0**
## What 
Upgrade API to v19.0. This PR incorporates contributor changes: https://github.com/airbytehq/airbyte/pull/35433

## 🚨 User Impact 🚨
Schema breaking change for stream `page`.

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

File: docs/integrations/sources/facebook-pages-migrations.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Facebook Pages Migration Guide
+
+## Upgrading to 1.0.0
+
+:::note
+This change is only breaking if you are syncing stream `page`.
```
Comment (katmarkham) [inline docs/integrations/sources/facebook-pages-migrations.md:6]: Page

File: airbyte-integrations/connectors/source-facebook-pages/metadata.yaml [hunk 2]
```diff
@@ -5,27 +5,38 @@ data:
...
+  remoteRegistries:
+    pypi:
+      enabled: true
+      packageName: airbyte-source-facebook-pages
   supportLevel: community
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: >-
+          This version updates the API version from v15 to v19. Deprecated fields have been removed and new fields have been added for the stream `page`.
```
Comment (katmarkham) [inline airbyte-integrations/connectors/source-facebook-pages/metadata.yaml:35]: Page

File: docs/integrations/sources/facebook-pages-migrations.md (hunk 1 above)
Comment (katmarkham) [reply docs/integrations/sources/facebook-pages-migrations.md:6]: we need to tell them what fields are disappearing. or if any of them are being replaced with a different name

**Review by katmarkham — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-facebook-pages/metadata.yaml (hunk 2 above)
Comment (tolik0) [reply airbyte-integrations/connectors/source-facebook-pages/metadata.yaml]: Fixed

File: docs/integrations/sources/facebook-pages-migrations.md (hunk 1 above)
Comment (tolik0) [reply docs/integrations/sources/facebook-pages-migrations.md]: Fixed

File: docs/integrations/sources/facebook-pages-migrations.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Facebook Pages Migration Guide
+
+## Upgrading to 1.0.0
+
+:::note
+This change is only breaking if you are syncing stream `Page`.
+:::
+
+This version brings an updated schema for the `v19.0` API version of the `Page` stream. The `messenger_ads_default_page_welcome_message` field has been deleted, and a few new fields have been added.
```
Comment (katmarkham) [inline docs/integrations/sources/facebook-pages-migrations.md:9]: please specify what's been added.

**Review by katmarkham — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/facebook-pages-migrations.md (hunk 3 above)
Comment (tolik0) [reply docs/integrations/sources/facebook-pages-migrations.md]: Added

**Review by katmarkham — APPROVED** [review]

**Pull request merged**